### PR TITLE
changed survey titles, to indicate feature

### PR DESCRIPTION
**Pull request by shahi645**
### What is the context of this PR?
just changed titles
### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).

### CODE REVIEW

File: data/en/test_sum_equal_or_less_validation_against_total.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "schema_version": "0.0.1",
     "data_version": "0.0.2",
     "survey_id": "0",
-    "title": "Grouped validation against total test survey",
+    "title": "Grouped sum of answers, in a single question must be less than or equal to a given total - test survey",
```
Comment (jonnyshaw89) [inline data/en/test_sum_equal_or_less_validation_against_total.json]: Very wordy

Comment (shahi645) [reply data/en/test_sum_equal_or_less_validation_against_total.json]: what would you like my friend?

File: data/en/test_sum_equal_or_less_validation_against_total.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "schema_version": "0.0.1",
     "data_version": "0.0.2",
     "survey_id": "0",
-    "title": "Grouped validation against total test survey",
+    "title": "A test schema for LESS THAN or EQUAL validation of a sum of answers against a given total",
     "theme": "default",
```
Comment (MebinAbraham) [inline data/en/test_sum_equal_or_less_validation_against_total.json]: Bit picky but not sure if that is still clear. What about:
`A schema to test a sum of answers are Less Than or Equal to a given total`?

**Review by MebinAbraham — APPROVED** [review]

**Review by MIslam92 — APPROVED** [review]

**Comment by shahi645** [comment]
Closed due to duplication

**Pull request closed without merging**